specification/indices/_types/DataStream.ts

```diff
@@ -32,7 +32,11 @@ import { DataStreamLifecycleWithRollover } from '@indices/_types/DataStreamLifec
 enum ManagedBy {
   /** @codegen_name ilm */
   'Index Lifecycle Management',
-  /** @codegen_name datastream */
+  /**
```

Comment:
Thinking further, this note should be a regular code comment aimed at API spec maintainers and not a JSDoc comment aimed at users (JSDoc comments end up in client libraries (e.g. Javadoc) and API documentation).

Something like:
```ts
/** codegen_name datastream */
// This should have been...
```